plasmapy/particles/tests/test_ionization_state.py

```diff
@@ -724,3 +722,14 @@ def test_iteration_with_nested_iterator():
         for fraction2 in hydrogen:
             i += 1
     assert i == 4
+
+
+@pytest.mark.xfail
+def test_electrons_can_be_put_into_ionization_state():
+    electrons = hydrogen = IonizationState("e-", n_elem=1e20 * u.m ** -3, T_e=10 * u.eV)
```

Comment:
I'm a tad confused about this test.  Is the idea to allow cases where the electron density isn't calculated from the ionization state, but rather given by the user without making the assumption of quasineutrality?

Reply:
Oh... yeah. Remember how I said this would be the non-controversial part? This slipped through.